packages/web-components/src/menu-item/menu-item.ts

```diff
@@ -0,0 +1,28 @@
+import { FASTMenuItem } from '@microsoft/fast-foundation';
+import { attr } from '@microsoft/fast-element';
+
+/**
+ * The base class used for constructing a fluent-menu-item custom element
+ * @public
+ */
+export class MenuItem extends FASTMenuItem {
+  /**
+   * Sets menu item styles for content alignment when checkboxes and radios are present
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: checkmark
+   */
+  @attr({ mode: 'boolean' })
+  public checkmark: boolean = false;
```

Comment:
This is odd to me - why is this named checkmark?

Reply:
Do you need an attribute to tell you when these are present considering space needs to be reserved for both `menuitemradio` and `menuitemcheckbox` instances?

Reply:
Similarly to `icons` this provides a way to actually target and align content in MenuItems that are not a `radio` or `checkbox` but do appear in a `Menu` that contains a subset of radios or checkboxes. It's also put us closer in parity with the FUIRv9 implementation which has two `boolean` props `hasCheckmarks` and `hasIcons`  which do the same thing.